### PR TITLE
Add Kafka alerts

### PR DESCRIPTION
**Pull request by raihankhan**

### CODE REVIEW

File: apis/alerts/v1alpha1/kafka_alerts_types.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	api "x-helm.dev/apimachinery/apis/releases/v1alpha1"
+)
+
+const (
+	ResourceKindKafkaAlerts = "KafkaAlerts"
+	ResourceKafkaAlerts     = "Kafkaalerts"
+	ResourceKafkaAlertss    = "kafkaalertss"
+)
+
+// MongodbAlerts defines the schama for KubeDB Ops Manager Operator Installer.
```
Comment (ArnobKumarSaha) [inline apis/alerts/v1alpha1/kafka_alerts_types.go:31]: there are a bunch of `mongodb` inside the `kafka-alerts` folder also. Change them all.

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: apis/alerts/v1alpha1/kafka_alerts_types.go (hunk 1 above)
Comment (ArnobKumarSaha) [reply apis/alerts/v1alpha1/kafka_alerts_types.go:31]: https://github.com/appscode/alerts/actions/runs/8294818988/job/22700681927?pr=57#step:6:150

Remove `successfulJobsHistoryLimit` field

Comment (raihankhan) [reply apis/alerts/v1alpha1/kafka_alerts_types.go]: removed

File: charts/kafka-alerts/ticket-template.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+You can contact to AppsCode with the following things attached,
+- [Get the kafka object](#get-kafka)
+- [Describe kafka object](#describe-kafka)
+- Describe the StatefulSet object: If there are multiple sts, describe all of them. To do that, first
+  [List statefulsets under a kafka](#list-statefulsets-under-a-kafka). Then [Describe statefulsets](#describe-statefulsets) for each of the sts.
+- Describe the pods: If there are multiple pod, describe all of them. To do that, first
+  [List pods under a kafka](#list-pods-under-a-kafka). Then [Describe pods](#describe-pods) for each of the pod.
+- [Check the secret is created](#check-secret)
+- [Check Pod logs](#get-pod-logs).  If it is a sharded kafka, you also need to check the [mongos bootstrap container's logs](#mongos-bootstrap-logs).
+- [Operator logs](#operator-logs)
```
Comment (ArnobKumarSaha) [inline charts/kafka-alerts/ticket-template.md:31]: What to do for a topology-enabled kafka ?

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

**Review by ArnobKumarSaha — APPROVED** [review]

**Pull request merged**